hatch.toml

```
@@ -43,6 +43,22 @@ test_adaptors = "pytest --no-cov {args:test/integ} -vvv --numprocesses=1 -m adap
 [envs.e2e.scripts]
 test = "pytest --no-cov {args:test/*/e2e}"
 
+[envs.integ-ci]
+pre-install-commands = [
+  "pip install -r requirements-integ-testing.txt",
+  "pip install -r requirements-testing.txt"
+]
```

Same suggestion as aws-deadline/deadline-cloud-for-blender#296.


```suggestion
pre-install-commands = [
  "pip install -r requirements-integ-testing.txt -r requirements-testing.txt",
]
```

---

Done